### DISCUSSION TITLE
Why in 12.0.1 a beta version of styled-jsx installed?  `styled-jsx: 5.0.0-beta.3`

### DISCUSSION THREAD

**Discussion by javadmalek** [vercel/next.js#30574] [Help]
In version 12.0.1, a dependency `styled-jsx: 4.0.1` is replaced with `styled-jsx: 5.0.0-beta.3` which is breaking?

**Comment by zanettin** [comment]
thanks for raising this topic @javadmalek 🙏  just have seen, that on the client bundle `styled-jsx` is included even if it is not referenced by the app.

![Bildschirmfoto 2021-12-22 um 09 16 11](https://user-images.githubusercontent.com/3241476/147059078-c1e4bb0c-8183-493c-90e2-bda6c3566180.png)
